app/models/katello/concerns/location_extensions.rb

```diff
@@ -15,11 +15,11 @@ def set_default_overrides
       end
 
       def reset_settings
```

Comment:
This is only called from `after_save`, no need to handle the `before_save` behavior here (the new functions are already present in 5.1).

Reply:
all of the code that have both cases, are called, not always directly from both hooks, i didn't wrote it just to cover possibilities, but only on cases where it happened

Reply:
Where? I can't find any other place in the code where these functions are called:
```
2.5.1 ~/gits/katello (master) $ git grep reset_setting
app/models/katello/concerns/location_extensions.rb:        after_save :reset_settings
app/models/katello/concerns/location_extensions.rb:      def reset_settings
```
and the same is true for the other two.